internal/commands/secrets/output.go

```diff
@@ -1,6 +1,56 @@
 package secrets
 
+import (
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
 const (
-	headerID   = "ID"
-	headerName = "Name"
+	headerID      = "ID"
+	headerName    = "Name"
+	headerDeleted = "Deleted"
+	headerDetails = "Details"
+
+	secretDeleteMessage = "Deleted Secrets"
```

Comment:
is this used anywhere?

Reply:
The table displays some message before the actual table itself, so it's used there
It's called "message" so I'm not sure what to name it or if it should be a const

Reply:
we can remove the `const` and just leave the string value as an arg to `terminal.NewTable(...`